server/controllers/gift.controller.js

```diff
@@ -0,0 +1,121 @@
+'use strict';
+
+const Gift = require('../../config/db').Gift;
```

Comment:
Could you do something like following instead?
```js
const DB = require('../../config/db')
const Gift = DB.Gift
....
```
Now you requiring the same file 3 times which looks redundant.